src/themis/themis_portable_endian.h

```diff
@@ -17,6 +17,52 @@
 #ifndef THEMIS_PORTABLE_ENDIAN_H
 #define THEMIS_PORTABLE_ENDIAN_H
 
+#include <stdint.h>
+#include <string.h>
+
 #include <soter/soter_portable_endian.h>
 
+static inline uint8_t* write_uint32LE(uint8_t* buffer, uint32_t value)
+{
+    uint32_t encoded = htole32(value);
+    memmove(buffer, &encoded, sizeof(encoded));
+    return buffer + sizeof(encoded);
+}
+
+static inline uint32_t read_uint32LE(const uint8_t** buffer)
```

Comment:
as commented above, what about to rename these functions?
anyway cool helpers for library

Reply:
Also, I feel the difference between the read/write semantics is counterintuitive: the `write` returns "adjusted" stream, while the `read` - the result of the operation. I would either do
```
void write(**buf, val)
uint32 read(**buf)
```
or
```
*buf write(*buf, val)
*buf read(*buf, *val)
```

I personally prefer the 2nd variant as don't like double pointers.